lib/api/apiUtils/authorization/tagConditionKeys.js

```diff
@@ -61,7 +61,6 @@ function updateRequestContextsWithTags(request, requestContexts, apiMethod, log,
             return cb(err);
         }
         // FIXME introduced by CLDSRV-256, this syntax should be allowed by the linter
-        // eslint-disable-next-line no-restricted-syntax
         for (const rc of requestContexts) {
```

Comment:
instead of disabling the linter, we should either update the linter configuration (to allow this), or retry this in a conforming maner : `requestContext.each(rc -> { ... })`